cmd/drand-cli/control.go

```diff
@@ -128,6 +128,13 @@ func leadShareCmd(c *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("period given is invalid: %v", err)
 	}
+	catchupPeriod := time.Duration(0)
+	if c.IsSet(catchupPeriodFlag.Name) {
```

Comment:
Given the default value there shouldnt be the need to check if it is set or not, we can just parse it directly.